.github/workflows/publish-integration-openapi.yaml

```diff
@@ -0,0 +1,23 @@
+name: OpenAPI Integration Publisher
+
+on:
+  push:
+    branches:
+      - integration
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        name: Check out repository
+
+      - uses: readmeio/rdme@7.5.0
+        name: Sync Open Payments API spec
+        with:
+          rdme: openapi resource-server.yaml --workingDirectory=openapi --key=${{ secrets.README_INTEGRATION_API_KEY }} --id=${{ secrets.README_INTEGRATION_OP_API_ID }}
+
+      - uses: readmeio/rdme@7.5.0
+        name: Sync Auth Server API spec
+        with:
+          rdme: openapi auth-server.yaml --workingDirectory=openapi --key=${{ secrets.README_INTEGRATION_API_KEY }} --id=${{ secrets.README_INTEGRATION_AUTH_API_ID }}
```

Comment:
It might be possible to re-use these steps for both publishing jobs/workflows (main + integration)

Reply:
Looking into it